pkgs/applications/office/paperwork/src.nix

```diff
@@ -1,14 +1,40 @@
-{ fetchFromGitLab }:
+{
+  fetchFromGitLab,
+  stdenv,
+  fetchpatch,
+}:
 rec {
   version = "2.2.5";
-  src = fetchFromGitLab {
-    domain = "gitlab.gnome.org";
-    repo = "paperwork";
-    group = "World";
-    owner = "OpenPaperwork";
-    rev = version;
-    sha256 = "sha256-PRh0ohmPLwpM76qYfbExFqq4OK6Hm0fbdzrjXungSoY=";
-  };
+  src =
+    let
+      actual_src = fetchFromGitLab {
+        domain = "gitlab.gnome.org";
+        repo = "paperwork";
+        group = "World";
+        owner = "OpenPaperwork";
+        rev = version;
+        sha256 = "sha256-PRh0ohmPLwpM76qYfbExFqq4OK6Hm0fbdzrjXungSoY=";
+      };
+    in
+    stdenv.mkDerivation {
+      pname = "paperwork-patched-src";
+      inherit version;
+      src = actual_src;
+
+      patches = [
+        # fix installing translations
+        # remove on next release
```

Comment:
Btw. one thing that I personally do when something is only temporary is adding an assertion on the version so that I immediately get an eval error and can remove the patch right away.